src/filters/time/time.test.ts

```diff
@@ -0,0 +1,74 @@
+import { DateTime } from 'luxon';
+import { TestCaseSchema } from '../../serina.schema';
+import Time from './time';
+
+describe('Time', () => {
+    const mockTime = (hour, minute) => DateTime.utc()
+        .set({ hour, minute })
+        .startOf('minute')
+        .toJSDate();
+
+    const testCases: TestCaseSchema[] = [
```

Comment:
I think it is a bit of an overkill, because the regex is quite straightforward. Testing 8:30 vs 9:30 adds very little value to me, if you feel we could cover more edge cases then I'd be fine with that. I don't think looping every hour is worth it, especially then the questions becomes should we loop all the minutes too?

My philosophy is add enough test to cover the key edge cases, every time you find a bug. Fix it then add a test to prevent similar bugs coming up in the future.

Reply:
Aye, maybe later. For now I think these tests should be enough